Src/Plugins/Security/GitHubPatValidator.cs

```diff
@@ -13,10 +15,19 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
     internal class GitHubPatValidator : ValidatorBase
     {
         internal static GitHubPatValidator Instance;
+        internal static IRegex RegexEngine;
+
+        private const string PatExpression = "[0-9a-z]{40}";
```

Comment:
>PatExpression [](start = 29, length = 13)

this needs to be case-insensitive (?i), right?

Reply:
lower case only

---
In reply to: [569700983](https://github.com/microsoft/sarif-pattern-matcher/pull/161#discussion_r569700983) [](ancestors = 569700983)